spec/openapi.yaml

```diff
@@ -325,6 +325,19 @@ tags:
       platform and specify additional filtering criteria to narrow down data
       returned by the reports.
 
+  - name: Attachments
+    description: |-
+      The Attachments API allows upload, download and link binary data with entity.
+      Unlinked attachments will be deleted after 2 days of the last upload
+      Client is responsible for entity uid validation and removing attachments within entity removal process
+      The following attachment types are supported:
+      - **jobs**: attachements for jobs
+      - **issues**: attachments for issues
+      - **strings**: attachments for strings
+
+      **Attachments API Usage Limitations**:
+      * Maximum file upload size is 500MB.
+
   - name: Translation quality checks
     description: The translation quality check service is an API endpoint at which you can check the quality of your translations.
```

Comment:
Is this related to configuring quality check profiles? If so, suggested edit: "The translation quality check service is an API endpoint at which you can set quality check profiles for linguists to following when translating your content in Smartling."

Reply:
no, this is just surrounding next to modified code. it's untouched.